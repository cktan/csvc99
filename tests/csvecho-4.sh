# Test Case : Error case (more than one field) 
../csvecho "one|two" 2>&1
echo
