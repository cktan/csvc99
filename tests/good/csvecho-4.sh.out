nfield expected to be 1, but is 2 instead

